src/etc/inc/openvpn.auth-user.php

```diff
@@ -53,7 +53,7 @@
 	$common_name = getenv("common_name");
 }
 
-if (!$username || !$password) {
```

Comment:
Why change this here? It would alter existing behavior for non-LDAP logins unrelated to this change.

Reply:
Hi,
The idea is remove password checks from here and let *_backed() functions handle password controls. 

Please let me know if you would like me to revert.